src/ts/components/adclickoverlay.ts

```diff
@@ -10,39 +10,30 @@ export class AdClickOverlay extends ClickOverlay {
   configure(player: PlayerAPI, uimanager: UIInstanceManager): void {
     super.configure(player, uimanager);
 
-    let clickThroughUrl = <string>null;
-    let clickThroughEnabled = !player.getConfig().advertising
-      || !player.getConfig().advertising.hasOwnProperty('clickThroughEnabled')
-      || (player.getConfig().advertising as any).clickThroughEnabled;
+    let clickThroughCallback: () => void = null;
 
     player.on(player.exports.PlayerEvent.AdStarted, (event: AdEvent) => {
```

Comment:
as discussed offline pls add a comment why this works.

Reply:
What exactly is unclear?

Reply:
It was about how the `AdClickoverlay` (and other ad components) gets the `AdStarted` event because they get initialized when the first `AdStarted` event gets triggered.

Reply:
I see. This description does not belong in here. This component simply listens to the event and does not need to be aware nor care why it receives it.